pom.xml

```diff
@@ -1479,16 +1479,6 @@
 
   <repositories>
     <repository>
-      <id>jboss-snapshots</id>
-      <url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-    </repository>
-    <repository>
```

Comment:
As it's not required, I don't see a point in leaving it in. We typically don't rely on snapshot dependencies. 
If you think, it's important I can add it in.